.prettierrc

```diff
@@ -3,5 +3,13 @@
   "semi": false,
   "singleQuote": false,
   "tabWidth": 2,
-  "trailingComma": "es5"
+  "trailingComma": "es5",
+  "overrides": [
+    {
+      "files": ["**/api-node-helpers-docs.js"],
+      "options": {
+        "semi": true
```

Comment:
This is lame - but `documentation` package used to read jsdocs is just working weirdly without semicolons (for more complex cases like this at least)

Reply:
Is it possible to add comments this file? This will look quite mysterious to the next person that finds it.

Reply:
Yes, the normal non-jsdoc comments shouldn't affect jsdoc (but given this `semi` problem that I spent few hours trying to figure out why things don't work as expected I won't guarantee this before trying

Reply:
Oops sorry - I meant adding a comment to this file to say "`documentation` package used to read jsdocs is just working weirdly without semicolons (for more complex cases like this at least), see https://github.com/gatsbyjs/gatsby/pull/12087"

Reply:
oh - yeah, we can should be able to switch `.prettierrc` to `.prettierrc.js`, and then we will be able to add inline comments

Reply:
Added comment in https://github.com/gatsbyjs/gatsby/pull/12087/commits/ee7ede041dde7230114e15fcdd04adb2e3542a76